src/main/java/apod/ApodFrame.java

```diff
@@ -0,0 +1,129 @@
+package apod;
+
+import apod.service.AstronomyPicOfDayServiceFactory;
+import com.github.lgooddatepicker.components.DatePicker;
+
+import javax.swing.*;
+import javax.swing.border.Border;
+import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.awt.image.BufferedImage;
+import java.time.LocalDate;
+
+public class ApodFrame extends JFrame {
+
+    private final DatePicker datePicker;
+    private final JButton submit;
+    private JLabel photo;
+    private JLabel photoTitle;
+    private JTextArea description;
+    private JButton prevYear;
+    private JButton nextYear;
+
+    private JPanel photoPanel;
+    private JPanel topPanel;
+    private JPanel bottomPanel;
+    private JPanel leftPanel;
+    private JPanel rightPanel;
+
+
+    private final ApodPresenter presenter;
+
+
+    public ApodFrame() {
+
+        setTitle("Astronomy Picture of the Day");
+        setSize(300, 200);
+        setDefaultCloseOperation(EXIT_ON_CLOSE);
+
+        setLayout(new BorderLayout());
+
+        topPanel = new JPanel();
+        bottomPanel = new JPanel();
+        bottomPanel.setLayout(new BoxLayout(bottomPanel, BoxLayout.PAGE_AXIS));
+        photoPanel = new JPanel(new FlowLayout());
+        leftPanel = new JPanel(new FlowLayout());
+        rightPanel = new JPanel(new FlowLayout());
+        add(topPanel, BorderLayout.NORTH);
+        add(bottomPanel, BorderLayout.SOUTH);
+        add(photoPanel, BorderLayout.CENTER);
+        add(leftPanel, BorderLayout.WEST);
+        add(rightPanel, BorderLayout.EAST);
+
+
+        datePicker = new DatePicker();
+        topPanel.add(datePicker);
+
+        submit = new JButton("Enter Date");
+        submit.addActionListener(this::onSubmitClick);
+        topPanel.add(submit);
```

Comment:
If you click the button without first picking a date, it doesn't work.
See if if you can populate the DatePicker with the current date and have it display today's image automatically without having to click "Enter Date"